app/lib/cms/database_file_system.rb

```diff
@@ -42,8 +42,8 @@ def partial_from_filesystem(path, context)
   private
 
   class LiquidPartialRenderer < ActionView::PartialRenderer
-    def initialize(*)
-      super
+    def initialize(lookup_context, options = {})
+      super(lookup_context, options)
```

Comment:
`ActionView::PartialRenderer` initializer has changed:

![Screenshot from 2024-03-12 15-36-34](https://github.com/3scale/porta/assets/1270328/29563dcb-5ddd-43a6-8e8c-116770753c13)

Reply:
I don't see a need for this change. We just forward the parameters.

Reply:
I am pretty sure I saw some error related to it... Otherwise I wouldn't go investigate the diffs :sweat_smile:. But now I don't remember what it was, and what you're saying makes sense. I'll check.

Reply:
OK, these failing tests made me remember :) 
```
ActionView::Template::Error: wrong number of arguments (given 1, expected 2)
    app/lib/cms/database_file_system.rb:46:in `initialize'
```

https://app.circleci.com/pipelines/github/3scale/porta/28249/workflows/ba755531-67f9-4f23-94ce-b6e3c672777d/jobs/316273

Here is how the initializer is called:
```
LiquidPartialRenderer.new(@lookup_context)
```

So, `super` expects 2 arguments and gets 1. `options = {}` avoid this problem.